cargo-typify/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+use std::path::PathBuf;
+
+use clap::Args;
+use color_eyre::eyre::{Context, Result};
+use schemars::schema::Schema;
+use typify::{TypeSpace, TypeSpaceSettings};
+
+/// A CLI for the `typify` crate that converts JSON Schema files to Rust code.
+#[derive(Args)]
+#[command(author, version, about)]
+pub struct CliArgs {
+    /// The input file to read from
+    pub input: PathBuf,
+
+    /// Whether to include a builder-style interface
+    #[arg(short, long, default_value = "false")]
+    pub builder: bool,
```

Comment:
sorry I didn't think of this earlier. I'd like to make the builder style the default. What do you think about two flags: `--builder` and `--positional` that are mutually exclusive?